### ISSUE TITLE
Bad [markdown][link) not detected 

### ISSUE THREAD

**Issue by dwlf** [ChrisWren/mdlint#11]
Bad [markdown][link) not detected 

```
$ echo "[two factor authentication][http://docs-stage.joyent.com/public-cloud/getting-started/2fa)" > link-test.md
$ mdlint link-test.md 
All markdown files passed linting
```
